packages/valory/skills/safe_deployment_abci/tests/test_contract_api.py

```diff
@@ -36,8 +36,25 @@
 from aea_ledger_ethereum import EthereumCrypto
 from aea_test_autonomy.configurations import ETHEREUM_KEY_DEPLOYER, get_key
 from aea_test_autonomy.docker.base import skip_docker_tests
+from aea_test_autonomy.fixture_helpers import hardhat_addr  # noqa: F401
+from aea_test_autonomy.fixture_helpers import hardhat_port  # noqa: F401
+from aea_test_autonomy.fixture_helpers import (  # noqa: F401
+    gnosis_safe_hardhat_scope_class,
+)
 
 from packages.valory.connections.ledger.connection import LedgerConnection
+from packages.valory.connections.ledger.tests.conftest import ganache_addr  # noqa: F401
+from packages.valory.connections.ledger.tests.conftest import ganache_port  # noqa: F401
+from packages.valory.connections.ledger.tests.conftest import key_pairs  # noqa: F401
+from packages.valory.connections.ledger.tests.conftest import ledger_api  # noqa: F401
+from packages.valory.connections.ledger.tests.conftest import owners  # noqa: F401
+from packages.valory.connections.ledger.tests.conftest import threshold  # noqa: F401
+from packages.valory.connections.ledger.tests.conftest import (  # noqa: F401
+    ethereum_testnet_config,
+    gnosis_safe_contract,
+    ledger_apis_connection,
+)
```

Comment:
Since we're not using conftest file, these all of the fixtures and their dependencies needs to be present in the context

Reply:
Just move it back to ledger connection!